### ISSUE TITLE
Starting a new project

### ISSUE THREAD

**Issue by dirad** [JEverhart383/astro-wordpress-starter#4]
Makes sure to use a theme that enables `Menus` and has at least one menu defined

The default `Twenty Twenty-Two` and `Twenty Twenty-Three` do not!

**Comment by djmtype** [comment]
@dirad Exactly. Any FSE themes like Twenty Twenty Two and above now support full site editing, therefore, they hide or have deprecated any trace of the old menu settings. I think it's a retrogression on Wordpress' part. It's also extremely absurd because what was just one click to manage the menus is now a full blown treasure hunt. 

While you can edit the menu items in the header template part, there is no official place where you "attach" or assign the location as PRIMARY. 

I had to enable Twenty-Twenty One using Jeff's Astro WP Starter. Otherwise, I ended up with an error.
